src/diffusers/loaders/single_file_model.py

```diff
@@ -208,7 +214,7 @@ def from_single_file(cls, pretrained_model_link_or_path_or_dict: Optional[str] =
                     )
                 )
 
-            if isinstance(original_config, str):
+            if isinstance(original_config, str) and "PixArt" not in class_name:
```

Comment:
PixArt doesn't have any official `yaml` like config like the SD ecosystem of the models do. This is the best I could think of while not introducing too many code changes. But I am open to other suggestions.